lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/wikiV2/V2DBOWikiPageDaoImpl.java

```diff
@@ -359,8 +359,6 @@ public V2WikiPage updateWikiPage(V2WikiPage wikiPage,
 			basicDao.createBatch(attachmentsToInsert);
 		}
 		
-		// Send the change message
-		transactionalMessenger.sendMessageAfterCommit(newDbo, ChangeType.UPDATE);
```

Comment:
We should not remove this, the WIKI object type has a SNS topic in the stack builder.